gst/nnstreamer/tensor_common.c

```diff
@@ -295,7 +295,7 @@ gst_tensors_info_validate (const GstTensorsInfo * info)
     return FALSE;
   }
 
-  for (i = 0; i < info->num_tensors; i++) {
```

Comment:
No, validation with the number of tensors. This will make all tensors-info is invalid.

Reply:
Right, my bad.